src/Status/Menu.tsx

```diff
@@ -0,0 +1,51 @@
+import { StatusElement } from './Status';
+
+function classNames( ...classes ) {
+    return classes.filter( Boolean ).join( ' ' );
+}
+
+const Menu = ( {
+    pages,
+    loading,
+    activePage,
+    onMenuClick,
+}: {
+    pages: StatusElement[];
+    loading: boolean;
+    activePage: string;
+    onMenuClick: ( page: string ) => void;
+} ): JSX.Element => {
+    return (
+        <aside className="py-6 px-2 sm:px-6 lg:py-0 lg:px-0 lg:col-span-3">
+            <nav className=" bg-white rounded-md">
+                { ! loading &&
+                    pages.map( ( item: StatusElement ) => {
```

Comment:
Pls handle `null` like `pages.map` to `(pages || []).map` 

CC: @shohag121  vai.